x/wasm/internal/keeper/staking_test.go

```diff
@@ -2,18 +2,20 @@ package keeper
 
 import (
 	"encoding/json"
+	"io/ioutil"
```

Comment:
minor: shuffled imports

Reply:
Yeah, my editor did this but I think the change is consistent with how the imports are normally ordered in Tendermint so I legt it.